packages/ontario-frontend-cli/create/create.js

```diff
@@ -44,12 +44,28 @@ async function createNewProject(answers, options) {
     projectDescription: answers.projectDescription,
   };
 
+  // If the user wants ESLint,
+  // will add the lint script to project's package.json file
+  if (answers.ESLint) {
+    conf.addScripts = conf.addScripts || {};
+    conf.addScripts.lint = `./node_modules/.bin/eslint '**/*.js'`; 
+    // ^ might need to add custom config and change the command here
+    // Where should we look for the config?
+    // eslint will probably by default look in the root directory
+    // might need to add specificity, firstly take from the root directory,
+    // secondly take from --config ./node_modules/eslint-config-ontario/.eslintrc.json
```

Comment:
Since this is happening during project scaffolding and creation, this could be a conditional in the package.json nunjucks template instead. Also, we don't need to do anything special for where it looks. The config file will be in the root of the project like normal.